src/main/groovy/com/bugsnag/android/gradle/BugsnagUploadNdkTask.groovy

```diff
@@ -50,7 +52,7 @@ class BugsnagUploadNdkTask extends BugsnagMultiPartUploadTask {
             project.logger.lifecycle("Found shared object file (${arch}) ${sharedObject}")
             sharedObjectFound = true
 
-            File outputFile = createSymbolsForSharedObject(sharedObject, arch)
+            File outputFile = generateSymbolsForSharedObject(sharedObject, arch)
```

Comment:
`create*` indicates a factory method in groovy - this has been renamed to avoid a warning.